settings.gradle

```diff
@@ -1,4 +1,2 @@
 rootProject.name = 'innholdstjeneste-backend-api'
-include 'src:main'
-findProject(':src:main')?.name = 'main'
```

Comment:
unsure why the earlier developers had this here. not sure what it does. maybe just remove it as you hva done here